### DISCUSSION TITLE
Power Amp with USB DAC no longer selectable

### DISCUSSION THREAD

**Discussion by sibezwo** [rern/rAudio#396] [Fixed Issues]
Hello Rern

Due to frequent Wi-Fi problems, I wrote the SD card with the current image 20210601 and installed the update 202106091.

Now I can no longer select my SMSL Q5 Pro as an output device.

Thomas

`# cat /etc/asound.conf`
`defaults.pcm.card 0`
`defaults.ctl.card 0`

`# aplay -l | grep ^card`
`card 0: Headphones [bcm2835 Headphones], device 0: bcm2835 Headphones [bcm2835 Headphones]`
`card 1: AMP [SMSL Q5 AMP], device 0: USB Audio [USB Audio]`

`# amixer scontrols`
`Simple mixer control 'Headphone',0`

**Comment by AndyFx12** [comment]
Please try a desktop-pc/network to select. Dropdown is broken on touchscreens at the moment.
Andy

**Comment by sibezwo** [comment]
Thank you for the new image 20210612.
Everything works fine again.

Thomas
